components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -20,12 +20,12 @@
           </tr>
         </tbody>
       </table>
-      <chef-tab-selector [value]="selectedTab" (change)="onTabChange($event)">
+      <chef-tab-selector [value]="tabValue" (change)="onSelectedTab($event)">
```

Comment:
I think we use `onTabChange` in other places as well . . . curious about the name change

Reply:
On auth pages we use `onSelectedTab`; that is precisely why I changed it.
Also, take a look at my commit message on the commit--I did a google search of how prevalent the two are. 😉